lib/hca/settings.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+module HCA
+  module Settings
+    CONFIG = Rails.application.config_for(:health_care_application).freeze
```

Comment:
I'm a bit out of date with ruby/rails loading stuff, but In production, wouldn't it only be loaded when the class is defined the first time?